flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
I would inquire as to why you excluded aarch64 users (RaspberryPi. Pine 64, etc) but then I saw the mention of Flutter upstream.